.github/workflows/recreate-linux-runners.yml

```diff
@@ -91,7 +91,7 @@ jobs:
           - linux-self-hosted-1
     steps:
       - name: Set up Cloud SDK
-        uses: google-github-actions/setup-gcloud@daadedc81d5f9d3c06d2c92f49202a3cc2b919ba # v0.2.1
+        uses: google-github-actions/setup-gcloud@77e7a554d41e2ee56fc945c52dfd3f33d12def9a # v2.1.4
```

Comment:
I think we're stuck on an older version here intentionally?

Reply:
Previous similar PR: 
* https://github.com/Homebrew/homebrew-core/pull/172749

Reply:
@carlocab Thanks, will ignore and move on. Note: it doesn't seem like a good idea to indefinitely sit on this old version. CC @Bo98 as it's been a year so the "soon" there may be stretching things a bit 😉